index.html

```diff
@@ -0,0 +1,200 @@
+    <!------------------------------HEADER END------------------------------------>
+    <!------------------------------MAIN START------------------------------------>
+    <main>
+      <section class="main-opening container">
+        <p class="intro">Bringing Total healthcare to the community</p>
+        <h1 class="topic">
+          Adopting Global Best Practices in
+          <br />
+          Community Pharmacy Practice
+        </h1>
+        <p class="description">
+          Attended by most major manufacturers, distributors and community
+          Pharmcists in the Pharmacy space, the ACPN conference provides a
+          focused and unique opportunity to learn, share best practices, network
+          with peers, and interact with other healthcare providers in the
+          pharmaceutical space. At the ACPN Conference, community Pharmacists
+          can learn how best to offer pharmaceutical care and gain insights on
+          more efficient drug therapy management.
+        </p>
+        <time datetime="2023-05-20T19:30">June 15, 2023 @ 9:00am</time>
+        <p class="venue">
+          Festival Hotel, along Amuwo Odofin, Festac town, Lagos State, Nigeria.
+        </p>
+      </section>
+    </main>
+    <!------------------------------MAIN  END------------------------------------->
+
+    <!------------------------------PROGRAMS STARTS------------------------------->
+    <section class="events">
+      <div class="container">
+        <h2>Main Program</h2>
+        <hr />
+        <div class="events-content">
+          <div class="content">
+            <div class="icon">
+              <i class="uil uil-analysis"></i>
+              <h3>Exhibition</h3>
+            </div>
+            <p>
+              40+ Pharmaceutical & FMCG companies have confirmed their
+              participation as exhibitors at the conference.
+            </p>
+          </div>
+          <div class="content">
+            <div class="icon">
+              <i class="uil uil-tv-retro"></i>
+              <h3>Excursion</h3>
+            </div>
+            <p>
+              Explore the vast Historic collections at the national Museum, Walk
+              through a range of art works at the Nike Art Garlary
+            </p>
+          </div>
+          <div class="content">
+            <div class="icon">
+              <i class="uil uil-comment-alt-notes"></i>
+              <h3>Workshop</h3>
+            </div>
+            <p>
+              The mandatory continuing professional development program that
+              will be hosted in this year’s conference is focused on Vaccination
+              Training for Pharmacists.
+            </p>
+          </div>
+          <div class="content">
+            <div class="icon">
+              <i class="uil uil-presentation-edit"></i>
+              <h3>Novelty Match</h3>
+            </div>
+            <p>
+              There will be a Novelty Match between the different state arm of
```

Comment:
- [ ] You did make the change suggested bu the previous reviewer here 👏 , however, I think you introduced a new bug that has made a margin lap by the right slide of your mobile view.
-  kindly see to this that its fixed, as it does not give a great user experience, and neither does it do justice to the good job you've done so far 👍 

## Your new view 👍 
![image](https://user-images.githubusercontent.com/50990925/234152019-0c85861a-9b87-4d6a-b38a-41282db989ae.png)